styles/globals.scss

```diff
@@ -0,0 +1,207 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@400;600&display=swap');
+@import '@/components/responsive';
+
+/* color themes for dark and light modes, generated with Leonardo.
```

Comment:
@andrewebdev it looks like some of the generated color combinations don't work well with color contrast requirements for accessibility. We'll have to check for that when working on the final design.

Reply:
Agree - we can look at colour contrasts later in the process

Reply:
My apologies for the confusion here @jupiter007 , the colours we have in the styleguide right now are not meant to be the final colours. My understanding is that we will only have the final colours around Aug/Sept.

My apologies for not making it clear that these are only temporary colours so that we can just get on with building :sweat_smile:

Reply:
Oops. apologies for the duplicate comment here.